mnq/sns-instances-notification-system/subscriber-server/www/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Subscriber - SNS x Instances Tutorial</title>
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;600&display=swap" rel="stylesheet">
+    <style>
+        body {
```

Comment:
**suggestion(duplication):** lots of inlined CSS in these HTML files. You could perhaps just load Bootstrap CSS from a CDN in one or two lines (e.g. like in the [quick start](https://getbootstrap.com/docs/5.3/getting-started/introduction/#quick-start)).